scalr/eval_and_analysis_pipeline.py

```diff
@@ -167,11 +173,15 @@ def perform_downstream_anlaysis(self):
                 analyser, analysis_config = build_analyser(analysis_config)
                 downstream_analysis[i] = analysis_config
 
-                analysis = analyser.generate_analysis(model=self.model,
-                                                      test_data=self.test_data,
-                                                      test_dl=self.test_dl,
-                                                      dirpath=self.dirpath,
-                                                      **self.primary_analysis)
+                # Note: Not passing Model & DataLoader since it is assumed that
+                # model is trained on train data, so analysis by model should be
+                # on test data only
```

Comment:
WE are saying test Data only and we passing full Data. Is it right?

Reply:
Yes, the comment here states that model will only be passed when test_data is used. In this case, since this is not test, then model and dl are not passed, as shown in first line